dgraph/cmd/alpha/run_test.go

```diff
@@ -1080,8 +1080,9 @@ func TestListTypeSchemaChange(t *testing.T) {
 	q = `schema{}`
 	res, err = runGraphqlQuery(q)
 	require.NoError(t, err)
-	testutil.CompareJSON(t, testutil.GetFullSchemaHTTPResponse(testutil.
-		SchemaOptions{UserPreds: `{"predicate":"occupations","type":"string"}`}), res)
+	testutil.CompareJSON(t,
```

Comment:
This will fail lint test, it probably goes beyond 120 chars in a line. I was suggesting this:
```
testutil.CompareJSON(t, testutil.GetFullSchemaHTTPResponse(
		testutil.SchemaOptions{UserPreds: `{"predicate":"occupations","type":"string"}`}), res)
```

Reply:
soga